docs/compilers/CSharp/Compiler Breaking Changes - post VS2019.md

```diff
@@ -78,3 +78,11 @@ could be different than the one that compiler used to find.
     ```
 
 11. Previously, the language version was not checked for `this ref` and `this in` orderings of parameter modifiers. In *Visual Studio 2019 version 16.4* these orderings produce an error with langversion below 7.2. See https://github.com/dotnet/roslyn/issues/38486
+
+12. https://github.com/dotnet/roslyn/issues/10492 Under some circumstances, the compiler would accept an expression that does not obey the rules of the language grammar.  Examples include
+    - `()=>{} + c`
+    - `e is {} + c`
+    - `e is T t + c`
+    - `delegate {} + c`
+
+    These all have in common that the left operand is of looser precedence than the `+` operator, but the left operand does not end in an expression so it cannot "consume" the addition.  Such expressions will no longer be permitted in Visual Studio 2019 version 16.5 (Update 5) and later.
```

Comment:
>(Update 5) [](start = 267, length = 10)

We should reference full versions rather than particular updates (and it looks like the update # is incorrect anyway).

Reply:
What full version do you suggest here?

---
In reply to: [352902202](https://github.com/dotnet/roslyn/pull/40089#discussion_r352902202) [](ancestors = 352902202)

Reply:
My suggestion is "Visual Studio 2019 version 16.5" without the "update ..." qualifier.

---
In reply to: [353406094](https://github.com/dotnet/roslyn/pull/40089#discussion_r353406094) [](ancestors = 353406094,352902202)